Documentation/rust/quick-start.rst

```diff
@@ -98,7 +98,7 @@ the ``bindgen`` tool. A particular version is required.
 
 Install it via (note that this will download and build the tool from source)::
 
-	cargo install --locked --version $(scripts/min-tool-version.sh bindgen) bindgen
+	cargo install --locked --version $(scripts/min-tool-version.sh bindgen) bindgen-cli
```

Comment:
I also discovered this yesterday. Could you send this patch directly to the mailing list?

@ojeda probably should take this as a fix.

Reply:
This is https://rust-for-linux.zulipchat.com/#narrow/stream/291565-Help/topic/Build.20errors/near/365685550 (https://github.com/ojeda/linux/commit/698129c961b3ac38459d03ffdde7648175da5f91), i.e. I was going to fix it directly when we take the upgrade to 0.65.1.